docs/tutorials/td-tutorial.md

```diff
@@ -0,0 +1,224 @@
+# Send data securely to Arm Treasure Data
+
+There are two ways to send data securely from Mbed OS to Treasure Data:
+
+- HTTPS library - Send data directly to the Treasure Data REST API.
+- fluentd using fluent logger library - Send data to a hosted fluentd instance that aggregates and forwards the data on to your treasure data account.
```

Comment:
ah ha this is how this works---I think Fluentd should be init capped

Reply:
I was told by the inventor that its always lowercase, see the website : https://www.fluentd.org/ . 

Either way, your call